src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -128,8 +129,40 @@ export class ExposureNotificationService {
     this.exposureStatus.observe(status => {
       this.storage.setItem(EXPOSURE_STATUS, JSON.stringify(status));
     });
+
+    // DeviceEventEmitter.addListener('initiateExposureCheck', this.initiateExposureCheckEvent);
+    DeviceEventEmitter.addListener('executeExposureCheckEvent', this.executeExposureCheckEvent);
   }
 
+  initiateExposureCheckEvent = async () => {
+    log.debug({category: 'background', payload: 'initiateExposureCheckEvent'});
```

Comment:
While `payload` will accept a string, in this case, would be better as `message`